Author: Benoit Mortier
Description: put the FusionDirectory Debian directory in the apache alias

--- a/core/contrib/apache/fusiondirectory-apache.conf	2012-05-21 23:02:13.000000000 +0200
+++ b/core/contrib/apache/fusiondirectory-apache.conf	2012-05-21 23:02:24.000000000 +0200
@@ -1,5 +1,5 @@
 # Include FusionDirectory to your web service
-Alias /fusiondirectory /var/www/fusiondirectory/html
+Alias /fusiondirectory /usr/share/fusiondirectory/html
 
 <IfModule mod_php5.c>
 <Location /fusiondirectory>
